src/server/binary/binaryServerStarter.ts

```diff
@@ -413,3 +425,7 @@ async function acceptBinaryDownloadResponse(response: http.IncomingMessage): Pro
     response.pipe(serverBinaryFileStream);
   });
 }
+
+async function openProxyDocumentation(): Promise<void> {
```

Comment:
can we track the open proxy doc event?

Reply:
Okay, I've added a telemetry event for clicking the link. What do you think of the name of the event?

Reply:
I forgot to document the event in the USAGE_DATA.md